docs/blueprints/EP038.rst

```diff
@@ -0,0 +1,186 @@
+
+Use Case: Interdomain Link Up/Down Monitoring
+
+Scenario: In SDX, monitoring the status of interdomain links for connectivity and reliability is crucial. This use case involves setting up a system to detect link status changes (up/down) and notify consumers about these events through message queues.
+
+Components:
+Producer: Generates events based on link status changes.
+Consumer: Monitors link status by consuming messages from the appropriate queues.
+
+Implementation:
+    
+Link Status Queue Initialization:
+    
+Each interdomain link has its dedicated queue.
+Queues are either dynamically created or configured based on settings.
+
+Benefits:
+Real-time monitoring of interdomain link status.
+Scalable solution with dynamically created queues.
+Fault-tolerant design ensures persistent handling of link-down events.
+Flexibility in queue management allows dynamic addition or configuration based on settings.
+
+With its efficient producer-consumer relationship, the system orchestrates message handling for specific link-down events. This design ensures smooth communication, preventing message flooding and instilling confidence in its performance. Here's how it operates:
+
+Producer Advertises Link Down: When a link-down event occurs, the producer publishes a message indicating the link's status change to down.
+
+Consumer's Crucial Role in Link Status Monitoring: The consumer, a key player, diligently monitors the message queue for link-down events. However, if the consumer has not yet read the message or the link status hasn't changed, the producer refrains from continuously writing messages of the same status to the queue.
+
+Preventing Message Flooding: To prevent message flooding and conserve system resources, the producer only writes messages to the queue if the consumer has consumed the previous message or the link status has changed.
+
+Message continuity is based on consumer activity; if the consumer has not acknowledged or processed the previous message, the producer waits for them to read it before publishing another message with the same status. In the same way, if the link status changes, the producer updates the message accordingly. 
+
+Publisher confirmations play a vital role in ensuring the reliability of message delivery. The mq_producer NApp gains confidence in successfully processing and routing a message upon receiving acknowledgment from the message broker, which prevents the mq_producer from continuously adding redundant messages to the queue. This acknowledgment mechanism ensures the reliable processing and routing of messages. 
```

Comment:
> routing a message upon receiving acknowledgment from the message broker, which prevents the mq_producer from continuously adding redundant messages to the queue.

This is good, but let's specify which protocol attrs we're talking here. In a PR this will need to be done and then you can refer to it. As of now this is just high level speaking about it, I have no idea if you're meaning to leverage the existing protocol flags to do it or roll out an implementation.